website/docs/getting-started/for-designers.md

```diff
@@ -61,7 +61,7 @@ Because the usage of Helios Foundations reinforces consistency across the produc
 
 3. Select the style you wish to apply from the list of options. Ensure you choose styles within the **HDS - Product Foundations** library.
 
-![Applying styles to a text layer](/assets/getting-started/designers/apply-text-style.png =784x*)
+![Applying styles to a text layer](/assets/getting-started/designers/apply-text-style.png =784x\*)
```

Comment:
[blocker] I don't know if this was done manually or by the page reformatter, but this I _think_ may break the special syntax we're using to declare the image size.

Reply:
Reverted this